tests/homepage/addonList.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount, shallow } from 'enzyme';
+import  { AddonList }  from '../../app/js/components/homePage/manageApps/addonList';
+
+
+describe('<AddonList />', () => {
+    const appList = [{
+        "version": "1.0.0",
+        "name": "CohortBuilder",
+        "description": "modern refresh of the OpenMRS Cohort Builder tool",
+        "icons": {
+            "16": null,
+            "48": "/img/omrs-button.png",
+            "128": null
+        },
+        "developer": {
+            "url": "https://github.com/openmrs/openmrs-owa-cohortbuilder",
```

Comment:
What is cohortbuilder doing here? ☺️

Reply:
It's a test object I'm passing as props so I can test the behavior of the `assonList.jsx` component

Reply:
Are you testing the cohort builder or addonmanager?

Reply:
I'm testing the `addonList` component of Addon manager. The object I used there was only to have values passed down to the component during testing. It is for testing purpose alone and so is not part of the component code